lib/datasource/helm/common.ts

```diff
@@ -40,6 +41,19 @@ export function findSourceUrl(release: HelmRelease): RepoSource {
     }
   }
 
-  // fallback
-  return { sourceUrl: release.sources[0] };
+  // fallback: if neither home nor sources are a chart repo URL, use githubUrl (if present)
+  const firstSourceMatch = githubUrl.exec(release.sources[0]);
+  if (homeMatch?.groups.url && homeMatch?.groups.path) {
+    return {
+      sourceUrl: homeMatch.groups.url,
+      sourceDirectory: homeMatch.groups.path,
+    };
```

Comment:
When this case will happen?

Reply:
When the `homeUrl` is a valid Github URL (matches `githubUrl` regexp) but the repo name it's not a valid chart repo (according to the `chartRepo` regexp)